musicnet.py

```diff
@@ -84,22 +87,24 @@ def add_weight_norms(m):
     # train_set = MusicNet(args.root, type='train', trainsize=trainsize, preprocess=args.preprocess, normalize=True, window=frame_size, epoch_size=esize)
 
     # -- with data augmentation
-    train_set = MusicNet(args.root, type='train', trainsize=trainsize, preprocess=args.preprocess, 
-                        pitch_transform=5, jitter=.1,
```

Comment:
`pitch_transform` should be `pitch_shift`, right?

Reply:
yes, it should be `pitch_shift`, lousy manual version control...😳